example/viam_example_app/lib/screens/base.dart

```diff
@@ -1,15 +1,16 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_platform_widgets/flutter_platform_widgets.dart';
 import 'package:viam_sdk/viam_sdk.dart';
+import 'package:viam_sdk/widgets.dart';
 
 class BaseScreen extends StatelessWidget {
   final Base base;
   final ResourceName resourceName;
```

Comment:
Do you need resourceName still? is it being used anywhere?

Reply:
We don't. I left it in because all the other screens still have `ResourceName`. I can remove it.